### PR TITLE
Consumer health check command

### PR DESCRIPTION
**Pull request by Mulkave**
Provides `php artisan bowler:healthcheck:consumer {queue} [--consumers=1]` to check the number of connected consumers according to the expected number in `--consumers`.

### CODE REVIEW

File: tests/Console/Commands/ConsumerHealthCheckCommandTest.php [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+        $this->app->bind(Connection::class, function() use($mConnection) {
+            return $mConnection;
+        });
+
+        $code = $this->artisan('bowler:healthcheck:consumer', ['queueName' => $queueName, '--consumers' => 1]);
+
+        $this->assertEquals(1, $code);
+    }
+
+    public function test_healthcheck_with_queue_does_not_exist()
```
Comment (KinaneD) [inline tests/Console/Commands/ConsumerHealthCheckCommandTest.php]: This test is failing

File: README.md [hunk 2]
```diff
@@ -307,6 +307,17 @@ To do so, the default laravel exception handler normaly located in `app\Exceptio
 
 `ExceptionHandler::reportQueue(Exception $e, AMQPMessage $msg)` allows you to report errors as you wish. While providing the exception and the queue message itsef for maximum flexibility.
 
+### Health Checks
+
+Based on [this Reliability Guide](https://www.rabbitmq.com/reliability.html), Bowler figured that it would be beneficial to provide
+a tool to check the health of connected consumers and is provided through the `bowler:healthcheck:consumer` command.
+
+```
+php artisan bowler:healthcheck:consumer {queue} [--consumers=1]
```
Comment (KinaneD) [inline README.md]: Add the abbreviation

File: composer.json [hunk 3]
```diff
@@ -14,23 +14,36 @@
...
+		"php-amqplib/php-amqplib": "v2.6.1",
+        "illuminate/console": "5.*",
+        "illuminate/support": "5.*",
+		"mockery/mockery": "^1.0"
 	},
 
     "require-dev": {
-        "phpunit/phpunit": "^5.7",
-        "mockery/mockery": "^0.9.7",
-        "friendsofphp/php-cs-fixer":"^1.12"
```
Comment (KinaneD) [inline composer.json]: Shouldn't we keep fixer?

**Comment by Mulkave** [comment]
Ready for another round.

File: tests/Console/Commands/ConsumerHealthCheckCommandTest.php [hunk 4]
```diff
@@ -25,6 +26,7 @@ public function test_checking_consumer_successfully()
 
         $mConnection = M::mock(Connection::class);
         $mChannel = M::mock(AMQPChannel::class);
+        $mChannel::$PROTOCOL_CONSTANTS_CLASS = 'PhpAmqpLib\Wire\Constants091';
```
Comment (KinaneD) [inline tests/Console/Commands/ConsumerHealthCheckCommandTest.php]: Ya wailak

**Pull request merged**